modules/nlu/src/backend/bootstrap.ts

```diff
@@ -41,19 +41,17 @@ export async function bootStrap(bp: typeof sdk): Promise<NonBlockingNluApplicati
     )
   }
 
-  const { endpoint, authToken } = getNLUServerConfig(globalConfig.nluServer)
-  const stanClient = new Client(endpoint, authToken)
-
-  const modelPassword = '' // No need for password as Stan is protected by an auth token
-  const engine = new StanEngine(stanClient, modelPassword)
+  const languageSource = getNLUServerConfig(globalConfig.nluServer)
+  const stanClient = new Client(languageSource.endpoint, languageSource.authToken)
+  const engine = new StanEngine(stanClient, '') // No need for password as Stan is protected by an auth token
```

Comment:
will the calls made by this instance fail when running in with cloud configuration?

Reply:
no because the health check is authentification free but this instance needs to be used only for health check reasons.